### PR TITLE
Clientside only

### PR DESCRIPTION
**Pull request by tofumatt**
Turn Face Value into an entirely client-side app.

### CODE REVIEW

**Comment by tofumatt** [comment]
Hey @fwenzel or @ednapiranha, wanna take a quick look at this and make sure it's sane?

**Comment by tofumatt** [comment]
(I know the tests don't pass, I'm still working on that.)

**Comment by tofumatt** [comment]
Tests are failing on Travis but no locally; turns out Travis is being weird. MERGIE TIME!

File: public/manifest.webapp [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "name": "Matthew Riley MacPherson (tofumatt)",
     "url": "http://argent.lonelyvegan.com/"
   },
-  "launch_path": "/app",
+  "launch_path": "/",
```
Comment (fwenzel) [inline public/manifest.webapp:14]: for a packaged app to work you probably need to do `/index.html`. Did you try it out in the simulator?

**Pull request merged**